libvirt/utils_net.go

```diff
@@ -16,7 +16,7 @@ const (
 )
 
 // randomMACAddress returns a randomized MAC address
-// with libvirt prefix
+// with QEMU prefix
```

Comment:
can we have some documentation or something which can be a comment to refer to this?

Reply:
That will probably be hard :-) From what I have seen in the forums, it is a "known fact" but it is hard to find a trusted, official, source.

If you find one, feel free to add it in the comment.

Reply:
we can put an issue link of this github project. that could be  a doc hint @Bischoff